python/cudf/cudf/utils/ioutils.py

```diff
@@ -1012,6 +1012,24 @@
 doc_kafka_datasource = docfmt_partial(docstring=_docstring_kafka_datasource)
 
 
+_docstring_text_datasource = """
+Configuration object for a text Datasource
+
+Parameters
+----------
+filepath_or_buffer : str, path object, or file-like object
+    Either a path to a file (a `str`, `pathlib.Path`, or
+    `py._path.local.LocalPath`), URL (including http, ftp, and S3 locations),
+    or any object with a `read()` method (such as builtin `open()` file handler
+    function or `StringIO`).
+delimiter : string, default None, The delimiter that should be used
+    for splitting text chunks into seperate cudf column rows. Currently
+    only a single delimiter is supported.
+
```

Comment:
Can you include documentation of what is returned? And on that note, do we really want to return a `DataFrame` here? The underlying libcudf API splits a string into a column of substrings according to a delimiter. I'm not 100% sure, but I assume that given a file it treats the entire thing as a single large string (possibly containing newline/carriage return characters) and splits it along the provided delimiter, leaving other escape characters (like newlines) intact. As such, wouldn't returning a `Series` make more sense than a single columned `DataFrame`?

Reply:
Agreed. It does make more sense to just return a Series and I made that change.